cosmos/providers/dbt/core/operators/base.py

```diff
@@ -139,6 +126,8 @@ def __init__(
         dbt_ol_path = shutil.which("dbt-ol")
         if dbt_executable_path == "dbt" and shutil.which("dbt-ol"):
             self.dbt_executable_path = dbt_ol_path
+        elif dbt_executable_path == "dbt":
```

Comment:
How does this change relate to the profile refactor?
Let's introduce it in a separate PR.